MBBSEmu/Session/SocketSession.cs

```diff
@@ -57,6 +57,11 @@ public virtual void Send(byte[] dataToSend)
             {
                 _logger.Warn($"Channel {Channel}: Attempted to write on a disposed socket");
             }
+            catch (Exception ex)
+            {
+                _logger.Error($"Channel {Channel}: Error Sending Data");
+                _logger.Error(ex);
```

Comment:
should we also rethrow in these cases? Otherwise the exception is just swallowed and might not invoke future catch handlers